pmm/pmm2-server-autobuild.groovy

```diff
@@ -0,0 +1,176 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+pipeline {
+    environment {
+        DESTINATION = 'laboratory'
+    }
+    agent {
+        label 'large-amazon'
+    }
+    parameters {
+        string(
+            defaultValue: 'PMM-2.0',
+            description: 'Tag/Branch for percona-images repository',
+            name: 'GIT_BRANCH')
+    }
+    options {
+        skipDefaultCheckout()
+        disableConcurrentBuilds()
+    }
+    triggers {
+        upstream upstreamProjects: 'pmm2-submodules-rewind', threshold: hudson.model.Result.SUCCESS
+    }
+    stages {
+        stage('Prepare') {
+            steps {
+                installDocker()
+
+                git poll: true, branch: GIT_BRANCH, url: 'http://github.com/Percona-Lab/pmm-submodules'
+                sh '''
+                    git reset --hard
+                    sudo git clean -xdf
```

Comment:
Why `sudo`?

Reply:
as I remember it is inheritance from pmm-server-autobuild job

Reply:
Ok, but why? 😄 It shouldn't be needed.

Reply:
I will remove it and check if it will work, thanks